net/wfb-ng/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wfb-ng
+PKG_VERSION:=24.9.1
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/svpcom/wfb-ng.git
+PKG_SOURCE_VERSION:=a41cd378e7e23dd094d6c50f26c4faa8e64e5954
+PKG_MIRROR_HASH:=362b3d4262e78ae0885cc25cea34e553dc29e1941be47724ece796bd35220d88
```

Comment:
Why do we need to clone Git sources? We can use tarball.  :-)

Reply:
IMHO for review (or for compare changes between two different versions) direct link to git commit is more preferable than tarball

Reply:
Comparing changes between two differents versions, thats why the tagged releases or tags are for. Still, I dont see reason, why BFU or the end user should care about your commit hashes, they want to have version and of course. Having double versioning in the Makefile, its odd even for developers. The version, which is not tagged, then it checkout sources and specific commit, no, no. Thats wrong.

Choose properly which one and I think you want to use tagged releases. :-)

Reply:
@BKPepe Ok. I've replaced it with `PKG_SOURCE_VERSION:=wfb-ng-24.09-openwrt-preview` now